include/debug.h

```diff
@@ -0,0 +1,11 @@
+#ifndef GUARD_DEBUG_H
+#define GUARD_DEBUG_H
+
+#define TX_DEBUG_SYSTEM             TRUE    // Enables a overworld debug menu for changing flags, variables, giving pokemon and more, accessed by pressing R plus START.
+#define TX_DEBUG_SYSTEM_IN_MENU     FALSE   // Replaces the overworld debug menu button combination with a start menu entry (replaces the exit entry).
+
+
+void Debug_ShowMainMenu(void);
+
+
+#endif // GUARD_DEBUG_H
```

Comment:
Needs a new line so Unix based systems won't comply.

Reply:
working on a unix system and never got an error, but I added a newline for safety